blocks/abp-review/utils/sendHelixData.js

```diff
@@ -21,12 +23,18 @@ const sendHelixData = ({
   if (visitorId) {
     data.push({ name: 'VisitorId', value: visitorId });
   }
+
   if (page) {
     data.push({ name: 'Page', value: page });
   }
+
   if (timeSpentInSeconds) {
     data.push({ name: 'TimeSpentInSeconds', value: timeSpentInSeconds });
   }
+  const profileDetails = getLocalStorage('profile');
```

Comment:
Maybe it's a good idea of move the functionality of getting the profile details somewhere else and pass it as function argument to `sendHelixData` (similar to how comment, lang, rating, postUrl, visitorId, page, timeSpentInSeconds are being passed now)